### DISCUSSION TITLE
Outdated CUSTOM_BUILD.MD

### DISCUSSION THREAD

**Discussion by jzuniga10** [LeanerCloud/AutoSpotting#474] [Q&A]
Hello, 
I am working on creating a custom image due to the latest release of the terraform module but am having difficulty when installing the AutoSpotting project. The command uses `get` which prompts me to being deprecated as well as it does not install the `/src/` directory for the source code. It only installs the `/bin` and `/pkg` directories. Any insight on this is greatly appreciated thanks

**Comment by jzuniga10** [comment]
Ignore, used following command for MAC OS and it worked `export GO111MODULE=off`. don't forget to source in `bash_profile` if needed

**Reply by cristim** [reply]
Thanks for raising this issue!

Custom builds are community supported, I'm only supporting the builds available from the AWS marketplace and not actively maintaining or testing this document so can't support you with this.

If you find any issues with the documentation and figure out how to improve it please submit a pull request.
